notifications/views/actions.py

```diff
@@ -81,12 +82,16 @@ def create_person(self, **kwargs):
         return person
 
 
-class ActionsFGasesView(ActionsBaseView):
-    """ Handles FGases registry fetching.
+class ActionsBaseFCSView(ActionsBaseView):
+    """ Handles FCS registry fetching.
     """
+    company_path = settings.FGASES_COMPANY_PATH
+    person_path = settings.FGASES_PERSON_PATH
+    success_msg = ''
+    error_msg = ''
 
     def cleanup(self):
-        """ Delete all persons and companies fetched from FGases registry.
+        """ Delete all persons and companies fetched from FCS registry.
         """
         Person.objects.filter(
             company__group__code__in=FGASES_GROUP_CODE
```

Comment:
You need to override this function in ActionODSView to retrive Persons with ODS_GROUP_CODE